package.json

```diff
@@ -278,8 +278,8 @@
     "@grafana/prometheus": "workspace:*",
     "@grafana/runtime": "workspace:*",
     "@grafana/saga-icons": "workspace:*",
-    "@grafana/scenes": "6.1.4",
-    "@grafana/scenes-react": "6.1.4",
+    "@grafana/scenes": "6.0.3--canary.990.13366792126.0",
```

Comment:
Should this be here? Feels odd that we're downgrading to a pre-release version of both these packages.

Reply:
We need to do a release of scenes when the other PR is merged, so this will be fixed

Reply:
just pushed an updated canary scenes version, this should fix failing tests and help us test this before merging the scenes code